### ISSUE TITLE
 save the canvas without the PDF pages as images

### ISSUE THREAD

**Issue by khaledmndev** [RavishaHesh/PDFJsAnnotations#57]
is there a way to save the canvas without the PDF pages as images to json? To reduce the of the json output, this way i can load the PDF file alone and import the canvas from the json..

**Comment by groyk** [comment]
I think this is a great idea. This way it shoud be possible to save and reedit as well.

**Comment by RavishaHesh** [comment]
Yes, that's a great idea. thanks for the suggestion. I'll try to add that as soon as possible. can't tell an exact date though
